python/py-urwid_readline/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        rr- urwid_readline 0.15.1
+name                py-${name}
+revision            0
+
+license             MIT
+maintainers         nomaintainer
+
+description         readline text edit for urwid
+long_description    Text input widget for urwid that supports readline shortcuts
+
+homepage            https://urwid.org/
+
+checksums           rmd160  2093bd172eec3f5d9f6ba660378b1d87440ed8cb \
+                    sha256  4bb0a4b72717c43cf249a82aeb392422f0047c1721fe476e0d18b05efafe5ced \
+                    size    8509
+
+python.versions     39 310 311 312 313
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
```

Comment:
the default is to use the `setuptools` PEP517 backend and thus there is no need to explicitly add this build dependency here